common/src/test/java/org/bonitasoft/console/common/server/themes/ThemeResourceServletTest.java

```diff
@@ -58,6 +71,46 @@ public void should_verify_authorisation_for_the_given_location_param() throws
         }
     }
 
+    @Test
+    public void should_use_infopath_if_location_param_is_not_defined() throws Exception {
```

Comment:
Do we still need those tests now that this behaviour does not belong to `ThemeResourceServlet`anymore?